index.js

```diff
@@ -69,9 +69,9 @@ const filters = {
     }
   },
   blockQuoteFilter: {
-    regex: /\`{3,}((?:[^\`{3,}]*)*)\`{3}/g,
```

Comment:
The `{3,}` part of the regex was inside a negated set `[^]`. So it matched anything that's not a backtick, a left curly brace, the digit 3 or a right curly brace. This part is not necessary, as we want the regex to stop matching when it encounters any backtick.

Additionally, the global flag needed to be removed from the regex used for the `test` call. Calling `test` on it had the side-effect of leaving a non-zero value on lastIndex.